auth/user_mgt.go

```diff
@@ -333,7 +358,8 @@ func (u *UserToUpdate) validatedRequest() (map[string]interface{}, error) {
 			if err != nil {
 				return nil, err
 			}
-			req["mfaInfo"] = mfaInfo
```

Comment:
Sorry for missing this, but was `mfaInfo` deprecated in the BE?

Reply:
I think it was always `mfa`. https://cloud.google.com/identity-platform/docs/reference/rest/v1/accounts/update
For ref [#530](https://github.com/firebase/firebase-admin-go/pull/530).